src/lib/AsyncTypeaheadInput.tsx

```diff
@@ -13,7 +13,7 @@ interface AsyncTypeaheadProps<T extends FieldValues> extends CommonInputProps<T>
 }
 
 const AsyncTypeaheadInput = <T extends FieldValues>(props: AsyncTypeaheadProps<T>) => {
-  const { label, helpText } = props;
+  const { disabled = false, label, helpText } = props;
```

Comment:
Is this needed? I think the default of `undefined` should work fine.

Reply:
Not needed, simply added it because we don't really use `undefined` as a value.
Should I remove it?

Reply:
@000-ooo-000 

Yes please remove it, the `undefined` value should be handled just fine by the component where it's passed to.

Reply:
Removed